Sources/TuistKit/Services/InitService.swift

```diff
@@ -92,10 +98,26 @@ class InitService {
 
         let directories = try templatesDirectoryLocator.templateDirectories(at: path)
         if let templateName = templateName {
-            guard
-                let templateDirectory = directories.first(where: { $0.basename == templateName })
-            else { throw InitServiceError.templateNotFound(templateName) }
-            let template = try templateLoader.loadTemplate(at: templateDirectory)
+            var template: Template
+            var tempPath: AbsolutePath?
+            if isURLFormattedString(templateName) {
+                let tempFolderName = AbsolutePath.current.removingLastComponent()
+                    .appending(.init("\(templateName)")).url.lastPathComponent
```

Comment:
Why are we removing the last component from `AbsolutePath`? Assuming that's the directory where the project will be created, we should take `AbsolutePath(path)` instead because that's what the user indicated through the `--path` argument, and create it relative to that.

Reply:
As we spoke at slack. the temp subfolder will have the name `Template`. As soon as i write the documentation, it will explain to user that need to call to the Template manifest -> `Template.swift`.